api_server.py

```
@@ -561,4 +570,5 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
+    # Sentinel: host="0.0.0.0" is required for containerization (B104).
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))  # nosec B104
```

Ruff (0.15.6)

[error] Possible binding to all interfaces

(S104)